Update User Guide and PPP (#32)
AY2223S2-CS2103-W16-3/tp: docs/UserGuide.md

### Adding a person: `add`

> Adds a person to the address book and adds existing event to this person if event index is specified

Reviewer: ```suggestion
Adds a person to the address book and adds existing event to this person if event index is specified.
```

Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [ev/EVENT_INDEX]…​`

* The event index refers to the index number shown in the displayed event list.
* The event index **must be a positive integer** 1, 2, 3, …​

<div markdown="span" class="alert alert-primary">:bulb: **Tip:**

> A person can have any numbers of event index (including 0)

Reviewer: ```suggestion
A person can have any number of events (including 0)
```

</div>

Examples:

> * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`adds person to the address book

Reviewer: ```suggestion
* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` Adds person to the address book.
```

> * `add n/Alex Yeoh p/89028392 e/alex@email.com a/Blk 142 Apple Street 23 ev/1` adds 1st event to the new person

Reviewer: ```suggestion
* `add n/Alex Yeoh p/89028392 e/alex@email.com a/Blk 142 Apple Street 23 ev/1` Adds 1st event to the new person `Alex Yeoh` in the address book.
```

>     added in the address book

Reviewer: ```suggestion
```
